src/files/url.js

```diff
@@ -49,6 +48,13 @@ class UrlFile extends BaseFile {
       jsonerrors: 1
     }
 
+    const metadata = getMetadataObject(this.settings)
+    if (metadata) {
+      $.each(metadata, (key, value) => {
+        data[`metadata[${key}]`] = String(value)
```

Comment:
Please ensure no escape required

Reply:
Our backend ignores keys containing square brackets. Waiting for Arsenii to resolve this question.

Reply:
There is no need to escape or duplicate validation logic. We'll read `Warning` header and print it to the console in the future. In the meantime, we'll make a detailed limitations description in the docs.